### PR TITLE
feat: replace video component

### PR DESCRIPTION
**Pull request by Arash217**
## What changes were made
The responsive video component content has been replaced with vue-dato-video component.

## How to test or check results
Go the en/blog/front-end-at-the-edge

## Checks
- [ ] All content model changes are done through [scripted migrations](../readme.md#scripted-migrations)
- [x] Appointed PR reviewers

### CODE REVIEW

File: src/components/responsive-video/responsive-video.vue [hunk 1]
```diff
@@ -1,248 +1,73 @@
...
-
-  @media (min-width: 1440px) {
-    .responsive-video {
-      width: 1440px;
-      margin-left: auto;
-      margin-right: auto;
-    }
-  }
+.responsive-video__caption-container {
+  margin-top: 0.5rem;
```
Comment (Siilwyn) [inline src/components/responsive-video/responsive-video.vue:55]: Let's use a spacing variable for this.

File: src/components/responsive-video/responsive-video.vue [hunk 2]
```diff
@@ -1,248 +1,73 @@
...
+}
+
+.responsive-video .app-icon {
+  position: absolute;
+  display: block;
+  top: 50%;
+  left: 50%;
+  width: 60px;
+  height: 60px;
+  transform: translate(-50%, -50%);
```
Comment (Siilwyn) [inline src/components/responsive-video/responsive-video.vue:66]: Is there a way to simplify the centering by using flexbox?

**Review by Siilwyn — CHANGES_REQUESTED** [review]
Nice improvement overall! Left some small remarks

**Review by Siilwyn — APPROVED** [review]

**Comment by Frankwarnaar** [comment]
Are we waiting for anything to merge this? @Arash217 @Siilwyn

**Pull request merged**